VideoWeb/VideoWeb/ClientApp/src/index.html

```diff
@@ -32,6 +32,7 @@
   </head>
 
   <body class="govuk-template__body app-body-class">
+    <script src="https://kit.fontawesome.com/757ee43f3e.js" crossorigin="anonymous"></script>
```

Comment:
Neil created this kit
a message from Neil

Just to let you know - this is a kit I created on my pro account (link above) not sure if we’d need to create one for the project on an HMCTS account or something?

Reply:
Raised as issue with Junior

Reply:
Included Lorenzo happy with it for now. Will be investigated further outside of this PR